sharness/t0000-sharness.t

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+test_description="Show basic features of Sharness"
```

Comment:
do we need this sharness example?

Reply:
No, but it is just an example of what can be done. It can be removed but it doesn't hurt to keep it.

Reply:
I say let's definitely remove this #0 test which is just an example from Shareness and has nothing to do with DVC before considering merging.